zsh/PKGBUILD

```diff
@@ -1,39 +1,56 @@
-# Maintainer: Pierre Schmitz <pierre@archlinux.de>
-# PGO Version: Laio Seman <laio [at] ieee.org>
+# CachyOS Maintainer: Roice Young <insanelydefault@proton.me>
+# CachyOS Contributor: Laio Seman <laio [at] ieee.org>
+# Arch Linux Maintainer: Pierre Schmitz <pierre@archlinux.de>
 
 pkgbase=zsh
 pkgname=('zsh')
 pkgver=5.9
-pkgrel=4
+pkgrel=6
 arch=('x86_64')
 url='https://www.zsh.org/'
 license=('custom')
 makedepends=('pcre' 'libcap' 'gdbm' 'yodl' 'gcc')
-source=("https://www.zsh.org/pub/zsh-${pkgver}"{,-doc}".tar.xz"{,.asc}
-    '0001-50629-do-not-use-egrep-in-tests.patch'
-    'fix-autocompletion.patch'
-    'zprofile')
+source=("https://www.zsh.org/pub/zsh-${pkgver}"{,-doc}".tar.xz"
+        '0001-50629-do-not-use-egrep-in-tests.patch'
+        '0001-50641-use-int-main.patch'
```

Comment:
Can you provide remote patches instead of local ones? Would be more preferable if possible

Reply:
Local patches are also used from Arch

Reply:
we also local patches, but remote ones still more preferable)

Reply:
I don't know if that's possible considering zsh's development is through mailing list.

Reply:
Yeah we can add link to the commit from their github mirror or so

Reply:
It appears the patches to not cleanly apply if I grab the mirrored versions, is it possible arch is using rebased patches?

Reply:
I'll look into this in a bit.

Reply:
it appears by looking at the diff of the arch patches and the files are slightly different, is it possible that for example that the link here is the wrong way to get the patch? https://github.com/zsh-users/zsh/commit/4fc5dc0292acd77f17281f451774ba2ca4203026.patch

Reply:
It's not wrong. If it's slightly different then Arch must have rebased the patches so it applies on top of 5.9 instead of master which is what the patch is based on.

Reply:
I see. Patches are failing to apply in the changelog. The actual code itself patches cleanly. We can leave it as local patches thank you.